tasks/rpmbuild-test

```diff
@@ -25,6 +25,24 @@ if [ "${fed_branch}" = "master" ]; then
 fi
 mkdir ${RSYNC_BRANCH}
 mkdir repo
+# Write uuid to a lock file and store a backup
+uuidgen > file.lock
+cp file.lock uuid.saved
+while false; do
+     # Check if lock exists on remote server
+     while [[ $(rsync --dry-run -avz file.lock fedora-atomic@artifacts.ci.centos.org::fedora-atomic/${RSYNC_BRANCH}/repo/lockdir) != *"file.lock"* ]]; do
+          sleep 60
+     done
+     # Push lock file with uuid to remote server
+     rsync -avz file.lock fedora-atomic@artifacts.ci.centos.org::fedora-atomic/${RSYNC_BRANCH}/repo/lockdir/
```

Comment:
You should also add the following rsync args to prevent a race condition when transferring files to the remote
`--ignore-existing`